Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook

```diff
@@ -55,6 +55,27 @@
\n            }\r\n        }\r\n    },\r\n    {\r\n        \"name\": \"SAP Netweaver work process utilization\",\r\n        \"description\": \"Fired when work process utilization exceeds 90  percent for DIA and BTC work type.\",\r\n        \"author\": \"Microsoft\",\r\n        \"templateId\": \"sapnetweaver-workprocess-utilization\",\r\n        \"severity\": \"2\",\r\n        \"alertTemplate\": {\r\n            \"query\": \"SapNetweaver_ABAPGetWPTable_CL | where PROVIDER_INSTANCE_s == '{ProviderInstance}' | where Typ_s has 'BTC' or Typ_s has 'DIA' | summarize totalWP = count(), freeWP = countif(Status_s == 'Wait') by TimeGenerated, Typ_s, hostname_s, instanceNr_d, serverTimestamp_t | extend UtilizationPctWP = round(toreal(totalWP - freeWP) / toreal(totalWP) * 100, 3) | summarize AggregatedValue = max(UtilizationPctWP) by bin(TimeGenerated, 1m), Typ_s, hostname_s, instanceNr_d, serverTimestamp_t\",\r\n            \"thresholdOperator\": \"GreaterThan\",\r\n            \"defaultThreshold\": \"90\",\r\n            \"alertUnit\": \"percent\",\r\n            \"metricMeasurement\": {\r\n                \"thresholdOperator\": \"GreaterThan\",\r\n                \"threshold\": \"2\",\r\n                \"metricTriggerType\": \"Consecutive\",\r\n                \"metricColumn\": \"PROVIDER_INSTANCE_s\",\r\n                \"frequencyInMinutes\": \"5\",\r\n                \"timeWindowInMinutes\": \"10\"\r\n            }\r\n        }\r\n    },\r\n    {\r\n        \"name\": \"SAP Netweaver queue utilization\",\r\n        \"description\": \"Fired when queue utilization exceeds 70  percent for DIA and BTC work type.\",\r\n        \"author\": \"Microsoft\",\r\n        \"templateId\": \"sapnetweaver-queue-utilization\",\r\n        \"severity\": \"2\",\r\n        \"alertTemplate\": {\r\n            \"query\": \"let baseQuery = SapNetweaver_GetQueueStatistic_CL | where PROVIDER_INSTANCE_s == '{ProviderInstance}' | where Typ_s has 'DIA' or Typ_s has 'BTC'; let auxQuery = baseQuery | summarize QueueLimit=max(Max_d), LatestServerTimestamp = arg_max(serverTimestamp_t, Now_d, TimeGenerated) by hostname_s, instanceNr_d, Typ_s; baseQuery | summarize count() by Typ_s | join kind=inner auxQuery on Typ_s | extend QueueUtilization = iff(QueueLimit != 0, (toreal(Now_d) / toreal(QueueLimit)) * 100, 0.0) | summarize AggregatedValue = max(QueueUtilization) by bin(TimeGenerated, 1m), Typ_s, hostname_s, instanceNr_d, LatestServerTimestamp\",\r\n            \"thresholdOperator\": \"GreaterThan\",\r\n            \"defaultThreshold\": \"70\",\r\n            \"alertUnit\": \"percent\",\r\n            \"metricMeasurement\": {\r\n                \"thresholdOperator\": \"GreaterThan\",\r\n                \"threshold\": \"2\",\r\n                \"metricTriggerType\": \"Consecutive\",\r\n                \"metricColumn\": \"PROVIDER_INSTANCE_s\",\r\n                \"frequencyInMinutes\": \"5\",\r\n                \"timeWindowInMinutes\": \"10\"\r\n            }\r\n        }\r\n    }\r\n]",
```

Comment:
* SAP Netweaver service unavailability, SAP Netweaver instance unavailability, SAP Netweaver enqueue server unavailability
  * It looks like we may not want the customer to be able to set the threshold, if so, remove the alertUnit field so the customer will not be able to change it.
* SAP Netweaver work process utilization, SAP Netweaver queue utilization
  * We may want to allow customer to edit these, if so, the description should not say the default percentage